clients/docs/confluent-cli.rst

```diff
@@ -0,0 +1,256 @@
+
+
+Basic Producer and Consumer
+----------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+.. Should steps 1 below be under this  "Produce Records" section or before it?
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      kafka-topics --bootstrap-server `grep "^\s*bootstrap.server" $HOME/.confluent/java.config | tail -1` --command-config $HOME/.confluent/java.config --topic test1 --create --replication-factor 3 --partitions 6
+
+#. Run the `Confluent CLI
+   producer <https://docs.confluent.io/current/cli/command-reference/confluent-produce.html#cli-confluent-produce?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   writing messages to topic ``test1``, passing in arguments for:
+
+   -  ``--cloud``: write messages to the |ccloud| cluster specified in
+      ``$HOME/.confluent/java.config``
+   -  ``--property parse.key=true --property key.separator=,``: pass key and
+      value, separated by a comma
+
+   .. code-block:: bash
+
+      confluent local produce test1 -- --cloud --property parse.key=true --property key.separator=,
+
+#. At the ``>`` prompt, type a few messages, using a ``,`` as the separator
+   between the message key and value:
+
+   .. code-block:: bash
+
+       alice,{"count":0}
+       alice,{"count":1}
+       alice,{"count":2}
+
+#. When you are done, press ``Ctrl-D``.
+
+.. source code for producer?
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the `Confluent CLI
+   consumer
+   <https://docs.confluent.io/current/cli/command-reference/confluent-consume.html#cli-confluent-consume?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   reading messages from topic ``test1``, passing in additional arguments:
+
+   -  ``--cloud``: read messages from the |ccloud| cluster specified in
+      ``$HOME/.confluent/java.config``
+   -  ``--property print.key=true``: print key and value (by default, it only
+      prints value)
+   -  ``--from-beginning``: print all messages from the beginning of the topic
+
+   .. code-block:: bash
+
+      confluent local consume test1 -- --cloud --property print.key=true --from-beginning
+
+You should see the messages you typed in the previous step.
+.. Is "previous step" is referring to step 3 in the "Produce Records" section?
+
+.. code-block:: bash
+
+   alice   {"count":0}
+   alice   {"count":1}
+   alice   {"count":2}
+
+When you are done, press ``Ctrl-C``.
+
+#. View the :devx-examples:`consumer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/confluent-cli/confluent-cli-example.sh`.
+
+
+Avro And Confluent Cloud Schema Registry
+----------------------------------------
+
+.. include:: includes/client-example-schema-registry-3.rst
+
+.. Do we want to create a separate section, for steps 1 through 5, leave them as is
+   or put them under the "Produce Records" section?
+
```

Comment:
@ybyzek Do we want to create a separate section, for steps 1 through 5 below, leave them as is,
   or put them under the "Produce Records" section?

Reply:
To match what the other client languages have, let's leave it as is.

Reply:
Got it